simple-wp-helpdesk/admin/class-reporting.php

```
@@ -286,13 +286,18 @@ function swh_report_kpi_data() {
 		)
 	) )->found_posts;
 
-	$resolution     = swh_report_avg_resolution_time();
-	$first_response = swh_report_first_response_time();
+	$cached_resolution = get_transient( 'swh_report_avg_resolution_time' );
+	$resolution        = is_array( $cached_resolution ) ? $cached_resolution : swh_report_avg_resolution_time();
 
+	$cached_first_response = get_transient( 'swh_report_first_response_time' );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $cached_first_response. Keep variable name length under 20. (undefined)

(LongVariable)